sched/task/task_setup.c

```diff
@@ -630,8 +632,11 @@ static int nxtask_setup_stackargs(FAR struct task_tcb_s *tcb,
 
   stackargv[argc + 1] = NULL;
 
-  tcb->cmn.group->tg_info->ta_argc = argc;
-  tcb->cmn.group->tg_info->ta_argv = stackargv;
+  if (ttype != TCB_FLAG_TTYPE_KERNEL)
+    {
+      tcb->cmn.group->tg_info->ta_argc = argc;
```

Comment:
why not remvoe ta_argc and ta_argv fields? and always get argv from nxsched_get_stackargs.

Reply:
Seems group arguments are still needed to support `cat /proc/n/cmdline` use case, here `n` is the pid of the task.

Reply:
it's better to find the root cause and fix it before merging.